lib/index.js

```diff
@@ -96,30 +96,30 @@ var util = require('./common/util.js');
  *
  * @example
  * var gcloud = require('gcloud')({
- *   keyFilename: '/path/to/keyfile.json',
- *   projectId: 'my-project'
+ *   projectId: 'grape-spaceship-123',
+ *   keyFilename: '/path/to/keyfile.json'
```

Comment:
All examples synced to use projectId (grape-spaceship-123) -> keyFilename. This is consistent across our docs.